scss/sponsor.scss

```diff
@@ -0,0 +1,428 @@
+.contact,
```

Comment:
at the top of the stylesheet, can you add some padding to left and right of all the content. something like this should work
`.main {
     padding-left: 7%;
     padding-right: 7%
}`

Reply:
I feel like a 10% padding would look more better? Dunno, just that I was looking at last year's site and the figma and thought that a 10% would look better imo. Ive pushed the changes to this branch so just lmk what you think.